src/trace.rs

```diff
@@ -58,7 +58,11 @@ pub fn init(color: bool, json: bool, levels: &str, enable_datadog_tracing: bool)
     // This is a hidden and undocumented functionality.
     let metrics_layer_enabled = metrics_layer_enabled();
 
+    //let (tasks_layer, tasks_server) = console_subscriber::TasksLayer::new();
+    //tokio::spawn(tasks_server.serve());
+
     let subscriber = tracing_subscriber::registry::Registry::default()
+        //        .with(tasks_layer)
```

Comment:
Do we want to keep the console subscriber? Or drop this commented code?

Reply:
I'll remove it. I do think it ended up being handy, but I'd like to see it properly introduced in a separate PR.

Reply:
Resolved in 0f4aac626